java-client/session/src/main/java/io/deephaven/client/impl/ChannelHelper.java

```diff
@@ -1,37 +1,33 @@
 package io.deephaven.client.impl;
 
 import io.deephaven.uri.DeephavenTarget;
-import io.grpc.ManagedChannel;
+import io.grpc.ChannelCredentials;
+import io.grpc.Grpc;
+import io.grpc.InsecureChannelCredentials;
 import io.grpc.ManagedChannelBuilder;
+import io.grpc.TlsChannelCredentials;
 
-public class ChannelHelper {
+public final class ChannelHelper {
 
-    public static final int DEFAULT_TLS_PORT = 8080;
+    public static final int DEFAULT_TLS_PORT = 443;
 
-    public static final int DEFAULT_PLAINTEXT_PORT = 8080;
-
-    public static ManagedChannel channel(DeephavenTarget target) {
-        return channelBuilder(target).build();
-    }
+    public static final int DEFAULT_PLAINTEXT_PORT = 10000;
```

Comment:
Note: this was functionally the default before from the context of the examples (ConnectOptions.DEFAULT_PORT).